components/movementsensor/fake/movementsensor.go

```diff
@@ -111,3 +111,6 @@ func (f *MovementSensor) Close(ctx context.Context) error {
 
 // ReadFix returns the fix of a fake gps movementsensor.
 func (f *MovementSensor) ReadFix(ctx context.Context) (int, error) { return 1, nil }
+
+// ReadSatsInView returns the number of satellites in view.
+func (f *MovementSensor) ReadSatsInView(ctx context.Context) (int, error) { return 2, nil }
```

Comment:
If this and ReadFix aren't being used by anything using the fake sensor, let's just remove them.

Reply:
we need ReadFix and ReadSatsInView to define a movement sensor type.